README.md

```diff
@@ -104,6 +104,10 @@ custom:
       - AUTH0_CLIENT_SECRET
 ```
 
+> required: true|false (default false)
```

Comment:
Let's think of another name to not use up `required` for requiring specific env vars to be set (future feature).

Reply:
Brainstorming:

```yaml
required:
  file: true
  variables:
    - API_ENDPOINT
```

to support both file and variable requirements.